source/screens/App/screens/Home/screens/__tests__/Home.test.js

```diff
@@ -1,9 +1,9 @@
 import React from 'react'
 import ReactDOM from 'react-dom'
-import UserProfile from '../UserProfile'
+import Home from '../Home'
 
 it('renders without crashing', () => {
   const div = document.createElement('div')
-  ReactDOM.render(<UserProfile />, div)
+  ReactDOM.render(<Home />, div)
```

Comment:
Was wondering if you were not planning to use enzyme/shallow?

Reply:
These are actually just the tests from CRA, I need to rewrite all of them